### ISSUE TITLE
python3: calendar-cli without options returns weird error rather than help

### ISSUE THREAD

**Issue by tobixen** [tobixen/calendar-cli#59]
Ref #33, python2 is sunset now.

Apparently the subcommand stuff in argparse differs in python3

```
$ python2 ./calendar-cli 
usage: calendar-cli [-h] [--config-file FILE]
                    [--config-section CONFIG_SECTION] [--interactive-config]
                    [--version] [--nocaldav] [--icalendar]
                    [--timezone TIMEZONE] [--language LANGUAGE]
                    [--caldav-url URL] [--caldav-user USER]
                    [--caldav-pass PASS] [--caldav-proxy CALDAV_PROXY]
                    [--ssl-verify-cert SSL_VERIFY_CERT] [--debug-logging]
                    [--calendar-url CALENDAR_URL] [--ignoremethod]
                    {todo,journal,calendar} ...
calendar-cli: error: too few arguments

$ python3 ./calendar-cli
Traceback (most recent call last):
  File "./calendar-cli", line 925, in <module>
    main()
  File "./calendar-cli", line 922, in main
    ret = args.func(caldav_conn, args)
AttributeError: 'Namespace' object has no attribute 'func'
```

**Comment by tobixen** [comment]
TLDR: I'm pushing a fix now.

This wasn't quite easy.  Eventually, https://stackoverflow.com/a/22994500 explains that the "required" attribute must be set in python3.  However, when doing this I got a traceback ...

```
  File "/usr/lib/python3.8/argparse.py", line 2035, in _parse_known_args
    ', '.join(required_actions))
TypeError: sequence item 0: expected str instance, NoneType found
```

Eventually, I decided to do a parser.print_help() if func is not set - but then it doesn't print the help from the subparsers.  Seems like the `set_default(func=...)` was a trick to pass the subcommand to the args ... there should probably be better ways of doing that, but decided to do the same thing for passing print_help to args.  Hm.
